docs/docusaurus.config.js

```diff
@@ -10,18 +9,19 @@ const config = {
   favicon: "img/favicon.ico",
 
   // Set the production url of your site here
-  url: "https://research-data-services.org",
+  url: "https://github.com",
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: "/",
+  baseUrl: "sciebo-rds.github.io",
```

Comment:
[I think](https://docusaurus.io/docs/api/docusaurus-config#baseUrl) this should only be `/`, whilst `url` in L 12 should be `sciebo-rds.github.io`?